tests/test-war-integration/src/main/resources/logging.properties

```diff
@@ -1,5 +1,5 @@
 com.google.cloud.runtimes.level = FINE
-com.google.cloud.runtimes.handlers = com.google.cloud.logging.LoggingHandler
```

Comment:
I think it should be `com.google.cloud.runtimes.handler=`. Have you tried that?
You might still have to define `handlers` first. I'm not sure.

Also, can you file a documentation bug against google-cloud-java?